html/semantics/embedded-content/the-iframe-element/iframe-loading-lazy-history-pushState.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<title>History state change for iframe loading='lazy' before it is loaded: history.pushState</title>
+<iframe data-src="support/blank.htm?foo" src="support/blank.htm?src" loading="lazy"></iframe>
+<script>
+const iframe = document.querySelector('iframe');
+// Should have no effect on lazy-loading
+iframe.contentWindow.history.pushState(null, "", iframe.dataset.src);
+</script>
+<!-- Loading testharness.js here is intentional. -->
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+setup({single_test: true});
+onload = () => {
+  step_timeout(() => {
+    assert_equals(iframe.contentWindow.location.href, new URL("support/blank.htm?src", location.href).href);
+    done();
+  }, 1000);
+};
+</script>
+<!-- delay the load event -->
+<iframe src="support/blank.htm?src"></iframe>
```

Comment:
Why is this iframe important? Why do we need to delay the load event by as little amount of time as it takes for this unrelated iframe to load?

Reply:
It's there to improve test stability. Relying on only a timeout is not ideal.

Reply:
Removing as part of switching to iframe load event.